addons/core/functions/fnc_preparePositionCoordinates.sqf

```diff
@@ -27,9 +27,13 @@ private _isSpectating = _unit getVariable ["TFAR_forceSpectator",false];
 
 private _isolated_and_inside = false; //_isInVehicle && {_unit call TFAR_fnc_vehicleIsIsolatedAndInside};
 private _vehicle = "no"; //if (_isInVehicle) then {_unit call TFAR_fnc_vehicleId} else {"no"};
-if (!isNull (objectParent _unit) && !_isSpectating) then {//_isInVehicle
+private _isInVehicle = !isNull (objectParent _unit);
+if ((!(isNil {_unit getVariable "TFAR_vehicleIDOverride"}) || _isInVehicle) && !_isSpectating) then {//_isInVehicle
```

Comment:
check _isInVehicle first, and do the isNil in lazy eval, isNil is somewhat a bit more expensive.

Maybe its cheaper to not use isNil?
maybe default value something like ctrlNull and then isEqualTo ctrlNull.
Would need to benchmark that.

This function runs for every unit for (almost) every frame, it needs to be top performing